compiler/src/iree/compiler/Pipelines/Pipelines.cpp

```diff
@@ -191,7 +191,11 @@ void buildIREEPrecompileTransformPassPipeline(
       globalOptimizationOptions.aggressiveTransposePropagation;
   globalTransformOptions.outerDimConcat =
       globalOptimizationOptions.outerDimConcat;
+  // The pipeline option has higher priority.
   globalTransformOptions.dataTiling = globalOptimizationOptions.dataTiling;
+  if (pipelineOptions.dataTiling) {
+    globalTransformOptions.dataTiling = true;
+  }
```

Comment:
Feel free to ignore, this just seems cleaner to me:
```suggestion
globalTransformOptions.dataTiling = 
	globalOptimizationOptions.dataTiling | pipelineOptions.dataTiling;
```

Reply:
I like the original code better because it explicitly reflects the priority and the logic is similar to the other part (i.e., dispatch creation one).